site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -135,33 +135,26 @@ default, Polaris uses an in-memory metastore.
 > The default in-memory metastore is not suitable for production use, as it will lose all data
 > when the server is restarted; it is also unusable when multiple Polaris replicas are used.
 
-To use a durable metastore, you need to switch to the EclipseLink metastore, and provide your own
-`persistence.xml` file. This file contains details of the database used for metastore management and
-the connection settings. For more information, refer to the [metastore documentation]({{% ref
-"metastores" %}}).
+To use a durable metastore, you need to switch to the Relational JDBC backed metastore 
 
-Then, configure Polaris to use your metastore by setting the following properties:
+Configure the `polaris.persistence` section in your Polaris configuration file
+(`application.properties`) as follows:
 
-```properties
-polaris.persistence.type=eclipse-link
-polaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml
-polaris.persistence.eclipselink.persistence-unit=polaris
 ```
+polaris.persistence.type=relational-jdbc
 
-Where:
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=<your username>
+quarkus.datasource.password=<your password>
 
-- `polaris.persistence.type` indicates that we are using the EclipseLink metastore.
-- `polaris.persistence.eclipselink.configuration-file` is the path to the `persistence.xml` file.
-- `polaris.persistence.eclipselink.persistence-unit` is the name of the persistence unit to use (in
-  case the configuration file has many persistence units).
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/<database_name>
+```
 
-Typically, in Kubernetes, you would define the `persistence.xml` file as a `ConfigMap` and set the
-`polaris.persistence.eclipselink.configuration-file` property to the path of the mounted file in
-the container.
+The relational JDBC metastore is a Quarkus-managed datasource and only supports Postgres and H2 as of now.
+Please refer to the documentation here:
+[Configure data sources in Quarkus](https://quarkus.io/guides/datasource)
 
-> [!IMPORTANT]
-> Be sure to secure your metastore backend since it will be storing sensitive data and catalog
-> metadata.
```

Comment:
The fact that this warning is removed suggests that no sensitive data is stored in the metastore anymore.  Is it the case?

Reply:
Good call, we should probably keep this warning

Reply:
Agree, with you all, it was an oversight, we do store the principal authentication data